hbase-shell/src/test/ruby/shell/shell_test.rb

```diff
@@ -123,14 +124,35 @@ class FooC; end
 
     # check that at least one of the HBase constants is present while evaluating
     io = StringIO.new <<~EOF
-      ROWPREFIXFILTER
+      ROWPREFIXFILTER.dump
     EOF
     output = capture_stdout { @shell.eval_io(io) }
     assert_match(/"ROWPREFIXFILTER"/, output)
   end
 
   #-----------------------------------------------------------------------------
 
+  define_test 'Shell::Shell#exception_handler should hide traceback' do
+    class TestException < RuntimeError; end
+    hide_traceback = true
```

Comment:
why do you both define a variable `hide_traceback = true` and also pass `true` to the `exception_handler` -- it's redundant?

Reply:
Agreed. I removed the unused variables.